plugins/ltac/g_ltac.mlg

```diff
@@ -390,9 +390,8 @@ END
 {
 
 let rm_abstract = function
-  | TacAbstract (t,_) -> t, true, false
-  | TacSolve [TacAbstract (t,_)] -> TacSolve [t], true, true
-  | x -> x, false, false
+  | TacAbstract (t,_) -> t, true
```

Comment:
This loses the abstract detection in `par: solve [abstract bla]`

Reply:
oh right